src/search/components/dateTime.component.tsx

```diff
@@ -14,27 +14,22 @@ export const datesEqual = (date1: Date | null, date2: Date | null): boolean => {
   return date1 !== null && date2 !== null && isEqual(date1, date2);
 };
 
-export interface UpdateFilterParams {
+export interface VerifyAndUpdateDateParams {
   date: Date | null;
   prevDate: Date | null;
   otherDate: Date | null;
   fromDateOrToDateChanged: 'fromDate' | 'toDate';
   changeDate: (date: Date | null) => void;
 }
 
-export function updateFilter({
+export function verifyAndUpdateDate({
   date,
   prevDate,
   otherDate,
   fromDateOrToDateChanged,
   changeDate,
-}: UpdateFilterParams): void {
-  if (
-    date &&
-    isValid(date) &&
-    !datesEqual(date, otherDate) &&
```

Comment:
We've removed this check here - from what I can tell this stops us from having identical dates in the from and to picker. Is there a reason this was removed? I suppose this case is caught by the `isBefore` checks below

Reply:
Do you mean the `datesEqual` function? I think it's still there, no? Line 32

Reply:
We don't check `datesEqual(date,otherDate)` anymore

Reply:
Yeah I just checked, if `from` and `to` are the same then it's now a valid date range - this could be fine though as just checking we use `gte` and `lte` in the query to the API, so now we accept being able to search for a specific second.

Reply:
Ah sorry, I realised my mistake the second I wrote that. I removed that because I was trying to allow the user to search by two equal dates so they can see shots in a one-second time interval. But then I cancel that out with the `isBefore` below so clearly I didn't think straight there. But none of this seems to work anyway because I _can_ search by equal dates at the moment?

While testing this, I've also noticed another bug which I've found a fix for, which was that copy pasting dates into the fields didn't work. So with all of that said, I'm going to convert this to draft and work on it further

Reply:
Addressed in https://github.com/ral-facilities/operationsgateway/pull/138/commits/66b9877748bb3bd79cd73955c89e8cabf373b85b